spec.md

```diff
@@ -480,6 +480,16 @@ idling and transport include:
 }
 ```
 
+The "Skip" `"mop"` is the empty operation that indicates *do nothing*. It is used in place of operations that will
+have no effect on the machine state, such as the global phase operation.
+
+```json5
+{
+  "mop": "Skip",
+  "duration": None
```

Comment:
Under the spec, "duration" is optional. So should you have it here? Or are you trying to be more explicit?

Reply:
I was trying to be more explicit here. Do you imagine a scenario where we can have non-`None` duration for "Skip"?

Reply:
No, I would imagine we would have something else like Idle or Noop or something for non-zero duration. I personally lean towards not including "duration" as it adds more noise and if anything, perhaps there should be a check in validation for any non-zero values attached to it. So you can attach None/0.0 duration if you wish but that is only optional and will error if otherwise specified.

Reply:
Sounds good!

You also seem to suggest I should not be lazy and add explicit model validation for `"mop"`s. Let me try to do that within this PR.

Reply:
That is, a code gen or person should normally just provide `{"mop": "Skip"}` but if they really want to they can attach a zero equivalent duration.

Reply:
Done!